authentik/root/settings.py

```diff
@@ -211,7 +211,13 @@
 DJANGO_REDIS_SCAN_ITERSIZE = 1000
 DJANGO_REDIS_IGNORE_EXCEPTIONS = True
 DJANGO_REDIS_LOG_IGNORED_EXCEPTIONS = True
-SESSION_ENGINE = "django.contrib.sessions.backends.cache"
+match CONFIG.get("session_storage", "cache"):
+    case "cache":
+        SESSION_ENGINE = "django.contrib.sessions.backends.db"
+    case "db":
+        SESSION_ENGINE = "django.contrib.sessions.backends.db"
+    case _:
+        raise ValueError("Invalid session_storage setting, allowed values are db and cache")
```

Comment:
There's improperlyconfigured for this I think?

Reply:
yeah I didn't know if it makes sense to raise that from the settings module